shared_python/Args.py

```diff
@@ -11,15 +12,11 @@ def _load_args_from_file(filepath):
   props = {}
   sep = ':'
   with open(filepath, "rt") as f:
-    for line in f:
-      l = line.strip()
-      if l:
-        key_value = l.split(sep)
-        key = key_value[0].strip()
-        value = sep.join(key_value[1:]).strip().strip('"')
-        props[key] = value
-  return props
-
+    try:
+      x = yaml.safe_load(f)
```

Comment:
Is there a reason we can't return the result of `yaml.safe_load(f)` directly as `x` doesn't add much to the legibility here?